cmd/gossamer/config_test.go

```diff
@@ -1302,3 +1307,97 @@ func Test_setLogConfig(t *testing.T) {
 		})
 	}
 }
+
+// newTestGenesisRawFile returns a test genesis file using "gssmr" raw data
+func newTestGenesisRawFile(t *testing.T, cfg *dot.Config) (filename string) {
+	filename = filepath.Join(t.TempDir(), "genesis.json")
+
+	fp := utils.GetGssmrGenesisRawPath()
+
+	gssmrGen, err := genesis.NewGenesisFromJSONRaw(fp)
+	require.Nil(t, err)
+
+	gen := &genesis.Genesis{
+		Name:       cfg.Global.Name,
+		ID:         cfg.Global.ID,
+		Bootnodes:  cfg.Network.Bootnodes,
+		ProtocolID: cfg.Network.ProtocolID,
+		Genesis:    gssmrGen.GenesisFields(),
+	}
+
+	b, err := json.Marshal(gen)
+	require.Nil(t, err)
+
+	err = os.WriteFile(filename, b, os.ModePerm)
+	require.NoError(t, err)
+
+	return filename
+}
+
+// NewTestConfig returns a new test configuration using the provided basepath
+func NewTestConfig(t *testing.T) *dot.Config {
```

Comment:
this doesn't need to be exported since it's already in an `_test.go` file, same with below

Reply:
Good point, I've updated these.

Reply:
I agree, always unexport by default 👍

Although it doesn't hurt anyone either. For example generated mocks are exported even when written to `mock_blabla_test.go` files, but they don't pollute the Go API.